crates/re_space_view_spatial/src/visualizers/lines3d.rs

```diff
@@ -126,36 +127,50 @@ impl Lines3DVisualizer {
             }
         }
 
-        let mut line_builder = ent_context.shared_render_builders.lines();
-        let mut line_batch = line_builder
-            .batch("lines 3d")
-            .depth_offset(ent_context.depth_offset)
-            .world_from_obj(ent_context.world_from_entity)
-            .outline_mask_ids(ent_context.highlight.overall)
-            .picking_object_id(re_renderer::PickingLayerObjectId(ent_path.hash64()));
-
-        let mut bounding_box = macaw::BoundingBox::nothing();
+        // Putting all entities into the same builder would be nice, but determining the strip & vertex count ahead of time is likely too costly.
+        let mut line_builder = re_renderer::LineDrawableBuilder::new(
```

Comment:
we could perhaps name the constructor better

```suggestion
        let mut line_builder = re_renderer::LineDrawableBuilder::with_max_strips_and_vertices(
```